torch/csrc/utils/python_arg_parser.h

```diff
@@ -303,9 +303,9 @@ struct PythonArgs {
   inline std::optional<c10::DispatchKeySet> toDispatchKeySetOptional(int i);
 
  private:
-  at::Tensor tensor_slow(int i);
-  at::Scalar scalar_slow(int i);
-  at::Scalar scalar_slow(PyObject* arg);
+  TORCH_PYTHON_API at::Tensor tensor_slow(int i);
+  TORCH_PYTHON_API at::Scalar scalar_slow(int i);
+  TORCH_PYTHON_API at::Scalar scalar_slow(PyObject* arg);
```

Comment:
This feels wrong: we should not make private functions part of the API...

Reply:
See my PR message on the top. Those are called in (performance critical) inline functions in .h file.

Reply:
IMO two potential solutions: Add `TORCH_PYTHON_API` to the entire class, which will make all its methods symbols visible or make those methods public, rather than private to the class implementation

Reply:
Let me try the 2nd solution, which looks more aligned with the goal of reducing API surace.